sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/DirectoryAsyncClient.java

```diff
@@ -671,7 +683,7 @@ public String getShareSnapshotId() {
     }
 
     private Flux<Integer> nextPageForForceCloseHandles(DirectorysForceCloseHandlesResponse response, String handleId, boolean recursive) {
-        List<Integer> handleCount = Arrays.asList(response.deserializedHeaders().numberOfHandlesClosed());
+        List<Integer> handleCount = Collections.singletonList(response.deserializedHeaders().numberOfHandlesClosed());
```

Comment:
are we always expecting one single element as the numberOfHandlesClosed?  singletonList will limit us to add just element right?

Reply:
Yes, numberOfHandlesClosed is a single value always. https://docs.microsoft.com/en-us/rest/api/storageservices/force-close-handles#response